core/cap-0033.md

```diff
@@ -44,17 +45,17 @@ This proposal is aligned with the following Stellar Network Goal:
   create highly usable products.
 
 ## Abstract
-We introduce the relation is-sponsoring-future-reserves-for, in which an
-account (the sponsoring account) pays any reserve that another account (the
-sponsored account) would have to pay. This relation is initiated by
-`BeginSponsoringFutureReservesOp`, where the sponsoring account is the source
-account, and is terminated by `EndSponsoringFutureReserveOp`, where the sponsored
-account is the source account. Both operations must appear in a single
-transaction, which guarantees that both the sponsoring and sponsored accounts
-agree to every sponsorship. We also introduce `RevokeSponsorshipOp`, which can
-be used to modify the sponsorship of existing ledger entries. To support this,
-we add new extensions to `AccountEntry` and `LedgerEntry` which record
-pertinent information about sponsorships.
+We introduce the sponsoring-future-reserves relation, in which an account (the
+sponsoring account) pays any reserve that another account (the sponsored account)
+would have to pay. This relation is initiated by `BeginSponsoringFutureReservesOp`,
+where the sponsoring account is the source account, and is terminated by
+`EndSponsoringFutureReservesOp`, where the sponsored account is the source
```

Comment:
I was just reading this a minute ago while writing up more examples for the docs and thought it may be worth highlighting the difference between Begin/End here:

> This relation is initiated by `BeginSponsoringFutureReservesOp`, where the **sponsoring** account is the source account, and is terminated by `EndSponsoringFutureReservesOp`, where the **sponsored** account is the source account.

It's very easy to otherwise overlook that tiny _textual_ but large _conceptual_ difference.